src/api/public_api_pagopa.ts

```diff
@@ -0,0 +1,310 @@
+/* tslint:disable:object-literal-sort-keys */
+/* tslint:disable:no-duplicate-string */
```

Comment:
there's no need to commit this file if you generate it at build time

Reply:
yes don't commit autogenerated files (this one and the models generated from the api specs)